pkg/kubectl/cmd/cp_test.go

```diff
@@ -113,10 +113,14 @@ func TestGetPrefix(t *testing.T) {
 	}{
 		{
 			input:    "/foo/bar",
-			expected: "foo/bar",
```

Comment:
this is a pretty different result, and doesn't involve any backsteps in the path... why did this change?